paasta_tools/contrib/migrate_chronos_percent_format.py

```diff
@@ -0,0 +1,197 @@
+"""Script for migrating Chronos command string formats.
```

Comment:
But a shebang here for `#!/usr/bin/env python3.6` make it executable and dh-virtualenv should replace it with the right shebang to make it slightly easier to run.